librecomp/src/vi.cpp

```diff
@@ -17,6 +17,10 @@ extern "C" void osViBlack_recomp(uint8_t* rdram, recomp_context* ctx) {
     osViBlack((uint32_t)ctx->r4);
 }
 
+extern "C" void osViRepeatLine_recomp(uint8_t* rdram, recomp_context* ctx) {
+    osViRepeatLine((uint32_t)ctx->r4);
```

Comment:
I think new functions should use the `_arg` function instead of accessing the raw registers when possible
```suggestion
    osViRepeatLine(_arg<0, u8>(rdram, ctx));
```